applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_solver.py

```diff
@@ -110,7 +110,11 @@ def GetDefaultSettings(cls):
             "reform_dofs_at_each_step": false,
             "line_search": false,
             "compute_reactions": true,
-            "block_builder": true,
+            "block_builder" : true,
+            "consider_lagrange_multiplier_constraint_resolution" : "None",
```

Comment:
```suggestion
            "consider_lagrange_multiplier_constraint_resolution" : "none",
```

BTW those are very specialized settings, I think we should consider making the B&S settings a sub-parameter

Reply:
If you see https://github.com/KratosMultiphysics/Kratos/pull/3185, you will see that I separate the configuration for everything. I will comment there

Reply:
so you are planning to do it altogether?
Why not start now if you already know how it will be?
Otherwise it will be more difficult to adapt